empress/support_files/js/select-node-menu.js

```diff
@@ -491,7 +491,7 @@ define(["underscore", "util"], function (_, util) {
             var node = nodeKeys[i];
             var x = this.empress.getX(node);
             var y = this.empress.getY(node);
-            highlightedNodes.push(...[x, y, 0, 1, 0]);
+            highlightedNodes.push(...[x, y, 65280]);
```

Comment:
```suggestion
            // Add a green circle indicating the highlighted node(s)
            highlightedNodes.push(...[x, y, 65280]);
```

(I figure it would make sense to explain that this color is green, since now it's harder to look at a color in the code and see what color it represents)